fastestimator/op/tensorop/loss/focal_loss.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Any, Dict, Iterable, List, Tuple, TypeVar, Union
+
+import tensorflow as tf
+import torch
+
+from fastestimator.backend import focal_loss
```

Comment:
when we import within non-apphub FE code we should use the full path (fastestimator.backend._focal_loss import focal_loss)